resources/saas-boost-web.yaml

```diff
@@ -120,39 +120,6 @@ Resources:
         - ALLOW_REFRESH_TOKEN_AUTH
         - ALLOW_USER_SRP_AUTH
       GenerateSecret: false
-  ConsoleIdentityPool:
-    Type: AWS::Cognito::IdentityPool
-    Properties:
-      IdentityPoolName: !Sub sb-${Environment}-identities
-      AllowClassicFlow: true
-      AllowUnauthenticatedIdentities: false
-      CognitoIdentityProviders:
-        - ClientId: !Ref ConsoleUserPoolClient
-          ProviderName: !Sub cognito-idp.${AWS::Region}.amazonaws.com/${ConsoleUserPool}
-          ServerSideTokenCheck: true
-  ConsoleIdentityPoolAuthRole:
-    Type: AWS::IAM::Role
-    Properties:
-      RoleName: !Sub sb-${Environment}-cognito-auth-role-${AWS::Region}
-      Path: '/'
-      AssumeRolePolicyDocument:
-        Version: 2012-10-17
-        Statement:
-          - Effect: Allow
-            Principal: {"Federated": "cognito-identity.amazonaws.com"}
-            Action:
-              - sts:AssumeRoleWithWebIdentity
-            Condition:
-              StringEquals:
-                cognito-identity.amazonaws.com:aud:
-                  - !Ref ConsoleIdentityPool
-              ForAnyValue:StringLike: {"cognito-identity.amazonaws.com:amr": authenticated}
-  ConsoleIdentityPoolRoleAttachment:
-    Type: AWS::Cognito::IdentityPoolRoleAttachment
-    Properties:
-      IdentityPoolId: !Ref ConsoleIdentityPool
-      Roles:
-        authenticated: !GetAtt ConsoleIdentityPoolAuthRole.Arn
```

Comment:
Why are we safe to delete the identityPools? From your description it sounds like Cognito functionality should be unchanged with this commit: was this unnecessary for Cognito?

Reply:
Correct - we've never used the Identity Pool. Don't be fooled by the poor product naming, an Identity Pool is a completely different animal than a User Pool and is used for a different purpose (a purpose that we do not need in SaaS Boost).